java/java.lsp.server/vscode/package.json

```diff
@@ -372,15 +372,22 @@
 				"title": "Compile Workspace",
 				"category": "Java"
 			},
+			{
+				"command": "java.clean.workspace",
+				"title": "Clean Workspace",
+				"category": "Java"
+			},
 			{
 				"command": "java.workspace.new",
 				"title": "New from Template...",
-				"category": "Java"
+				"category": "Java",
+				"icon": "$(new-file)"
 			},
 			{
 				"command": "java.workspace.newproject",
 				"title": "New Project...",
-				"category": "Java"
+				"category": "Java",
+				"icon" : "$(new-folder)"
```

Comment:
What does `${new-folder}` mean?

Reply:
a reference to VS embedded icon. Feel free to suggest a (visually) better one.